### DISCUSSION TITLE
Script not finding available music on tidal

### DISCUSSION THREAD

**Discussion by thedinz** [RandomNinjaAtk/arr-scripts#136] [Q&A] [answered]
I have some popular artists that tidal is not able to find matches for and I cant figure out why, I open my tidal app and the artist is there and the albums I want as well yet tidal runs and finds no matches.

To most things lately in fact, it has not downloading anything in a bit and a lot of this seems to be readily available stuff. No errors in the log aside from saying not found or not a match.

**Comment by RandomNinjaAtk** [comment]
You need to provide examples and log output if you want this to be looked into further...

**Reply by thedinz** [reply]
<img width="981" alt="Screen Shot 2023-12-15 at 12 35 30 PM" src="https://github.com/RandomNinjaAtk/arr-scripts/assets/68015411/77ec36ca-47d9-4e01-b068-72b693a3abec">
<img width="1058" alt="Screen Shot 2023-12-15 at 12 36 36 PM" src="https://github.com/RandomNinjaAtk/arr-scripts/assets/68015411/7f9400a9-b13d-4309-acf2-9a4dc7ddc23e">

**Comment by thedinz** [comment]
Great and whats the best way and by that I mean do I delete any data from the not found folder and run again or just restart the container? When I restart it it just says 640 albums monitored no matches, if I delete the not found data it gets more in depth as it contacts tidal and tries to match. Also im in the logs folder, wow there is alot, which one is relevant to this case?

**Comment by thedinz** [comment]
logs

https://mega.nz/folder/4xIDmIza#8KAPmXcqQc47-zW4b3_2Tg

**Reply by RandomNinjaAtk** [reply]
I appreciate that you provided log data. However, I do not have the cycles to sift through that much data.... You need to provide the log data for a search of an album, that you expect it to find... A lot of times in the log data, it will tell you why it did not find a match....

A lot of the times, things fail because names don't match up, track number differences, songs failed to download and etc... The only way to know what happen is to see it in the logs.

**Comment by RandomNinjaAtk** [comment, marked as answer]
FYI: I setup a brand new instance (using default configuration provided by autoconfig) to attempt to replicate if this is a valid issue. I was not able to reproduce it (validated, using same artist and dl client). So there is nothing to be done, because what is happening in your case is either a user configuration error of sorts, or some other problem that is outside of the scope of this project...

**Reply by thedinz** [reply]
I appreciate you doing that, so is that to say that on your instance tidal found and downloaded tracks for that artist? Can't imagine what I have going on that would not let that work. I download from tidal all the time, just not this artist.

**Reply by RandomNinjaAtk** [reply]
Correct, it downloaded 119 tracks out of 199 possible tracks for that artist.

**Reply by thedinz** [reply]
wow, thats wild. Ok lets say I want to start over and setup tidal from scratch, are their instructions on that?

**Reply by RandomNinjaAtk** [reply]
If you stop the container, delete the `/config/extended` folder and all of its contents, and delete the `extended.conf` file, you will have effectively reset everything, when the container is restarted after...

keep in mind, you'll want to edit the config file with any changes you need to make...

**Comment by thedinz** [comment]
Ok man, I did that, but oddly tidal never asked me for my credentials or token again, and im seeing lots of this in the log, is this a permissions thing? is it not able to delete the file, also not sure why its trying to delete anything, this is the first run after reconfigure.

2023-12-21 14:52:45.1|Error|EventAggregator|MediaFileDeletionService failed while processing [TrackFileDeletedEvent]

[v2.0.7.3849] System.IO.DirectoryNotFoundException: Could not find a part of the path '/music/Dorian (US rapper+producer Alton Dorian Clark)'.
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)
   at System.IO.Enumeration.FileSystemEnumerator`1.Init()
   at System.IO.Enumeration.FileSystemEnumerableFactory.UserFiles(String directory, String expression, EnumerationOptions options)
   at System.IO.Directory.InternalEnumeratePaths(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)
   at NzbDrone.Core.MediaFiles.MediaFileDeletionService.Handle(TrackFileDeletedEvent message) in ./Lidarr.Core/MediaFiles/MediaFileDeletionService.cs:line 198
   at NzbDrone.Core.Messaging.Events.EventAggregator.PublishEvent[TEvent](TEvent event)

**Reply by thedinz** [reply]
However aside from that error, would love to fix it if possible, tidal is now downloading again, very odd that I needed to reset it like that?
